packages/adapter/tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "baseUrl": "./",
-    "outDir": "dist"
+    "outDir": "dist",
+    // Enable at root tsconfig.json once all packages have been fixed
```

Comment:
Maybe just add `TODO: Set to true once all of the packages have migrated to strict mode` to the root `tsconfig.json` with `strict: false` instead of a comment to every pckg?

Reply:
Yeah good idea